sdk/core/azure-core/src/http/curl/curl.cpp

```diff
@@ -21,15 +21,11 @@ std::unique_ptr<Response> CurlTransport::Send(Context& context, Request& request
     {
       case CURLE_COULDNT_RESOLVE_HOST:
       {
-        throw Azure::Core::Http::CouldNotResolveHostException();
-      }
-      case CURLE_WRITE_ERROR:
-      {
-        throw Azure::Core::Http::ErrorWhileWrittingResponse();
+        throw new Azure::Core::Http::CouldNotResolveHostException("Could not resolve host");
```

Comment:
Could you add the host we were looking for into the message to help a customer diagnose and fix this issue?